src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-tiering-form/rgw-bucket-tiering-form.component.ts

```diff
@@ -57,9 +57,29 @@ export class RgwBucketTieringFormComponent extends CdForm implements OnInit {
     this.rgwBucketService
       .getLifecycle(this.bucket.bucket, this.bucket.owner)
       .subscribe((lifecycle) => {
-        this.configuredLifecycle = lifecycle || { LifecycleConfiguration: { Rules: [] } };
+        if (lifecycle == null) {
```

Comment:
```suggestion
        if (lifecycle === null) {
```
This is coming up again and again. We should be careful, and avoid using `==` as a general rule. Thanks